samples/template_subghz/fsk.conf

```diff
@@ -5,9 +5,3 @@
 #
 
 CONFIG_SIDEWALK_LINK_MASK_FSK=y
-
-CONFIG_HW_CC3XX=y
-CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
-CONFIG_PSA_CRYPTO_DRIVER_OBERON=n
-
-CONFIG_SPIN_VALIDATE=n
```

Comment:
I though it was important for fsk to meet time restrictions. Please take special care about FSK tests